docs/doc_1_concepts.md

```diff
@@ -47,6 +47,10 @@ Use the `{Operation}()` method to perform modifications. For example:
  resp, r, err := sdk.ProjectsApi.CreateProjectInvitation(context.Background(), groupId, &groupInvitationRequest).Execute()
 ```
 
+## Experimental Tag
+
+Please note that some endpoints, denoted by the (experimental) tag in the operation description, have not undergone extensive end-to-end validation and may undergo changes in future releases.
+
 ## Example
```

Comment:
I would rephrase it little bit to incluide:

- Flag that reason why we have those experiemental is that SDK is constantly updated with the changes
- Flag the risks: Feel free to use it but you might get breaking changes. Some inconsistencies might also appear. 
```suggestion
## Example
```
```suggestion
## Example
```
- Link to the reference containing all